frontend/packages/console-dynamic-plugin-sdk/src/api/dynamic-core-api.ts

```diff
@@ -158,3 +160,14 @@ export const ResourceYAMLEditor: React.FC<ResourceYAMLEditorProps> = require('@c
  */
 export const ResourceEventStream: React.FC<ResourceEventStreamProps> = require('@console/internal/components/events')
   .WrappedResourceEventStream;
+
+export const usePrometheusPoll: UsePrometheusPoll = require('@console/internal/components/graphs/prometheus-poll-hook')
+  .usePrometheusPoll;
+
+const fromNow: FromNow = require('@console/internal/components/utils/datetime').fromNow;
+
+/** Various functions to humanize numbers, units or time wrapped to an object as a namespace */
+export const humanize = {
+  // More converter functions are expected to be added
```

Comment:
```suggestion

```

doesnt that apply for everything ? :)

Reply:
We have many utility functions to humanize values, I was thinking about namespacing them here.